tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html

```diff
@@ -407,10 +407,9 @@
       </div>
       <template is="dom-if" if="[[healthPillsFeatureEnabled]]">
         <div class="control-holder">
-          <paper-toggle-button
-            checked="{{healthPillsToggledOn}}"
-          ></paper-toggle-button>
-          <div class="title">Show health pills</div>
+          <paper-toggle-button checked="{{healthPillsToggledOn}}"
+            >Show health pills</paper-toggle-button
```

Comment:
our formatter sucks :) can you make sure this is proper?

Reply:
Prettier doesn’t render it as 

```html
<paper-toggle-button
  checked="{{healthPillsToggledOn}}"
  checked="{{healthPillsToggledOn}}"
>
  Show health pills
</paper-toggle-button>
```

by default because that’s technically a semantic change due to HTML
whitespace interpretation rules. If you write it like that, though, then
it will keep it that way. (My favorite feature of JSX may be that it
doesn’t have these inane whitespace rules.)

Reply:
I expected to see one of these two:

```html
<paper-toggle-button checked="{{healthPillsToggledOn}}"
  >Show health pills</paper-toggle-button>
```

```html
<paper-toggle-button
  checked="{{healthPillsToggledOn}}"
>Show health pills</paper-toggle-button>
```

Also, what you said is true but is also a variable of CSS rules. You can make all the whitespaces not render. As a formatter, it is unreasonable to expect to parse CSS and realize that but at least the `>` can be on the same line. :\

Reply:
@wchargin @stephanwlee  The formatting you are seeing is the output of `yarn fix-lint`.

Reply:
I just added `class="title"` to the paper-toggle-button. It's necessary to make the fonts consistent. The formatting is still a little weird. But the GUI looks okay. See the screenshot in the PR description.